tests/assets-pipeline.test.js

```diff
@@ -268,6 +271,16 @@ describe(__filename, () => {
         expect(html).toContain(ap.assetMap['assets/css/file1.css'].hashedPath);
       });
 
+      it('should have updated the default social image url', async () => {
+        const html = await fs.readFile(path.join(dest, 'index.html'), 'utf8');
+
+        const url = [
+          'https://example.com',
+          ap.assetMap['assets/octoamo-sm.png'].hashedPath,
+        ].join('/');
+        expect(html).toContain(`<meta property="og:image" content="${url}">`);
```

Comment:
It makes sense to write this so that it will be updated when the file changes. However, I wonder if it should do something to assert the path does contain a hash, or in other words ensure it has been transformed in some way. 

If the hashedPath for this URL was missing the hash for some reason, this test would still pass.